doc/man3/EVP_PKEY_gettable_params.pod

```diff
@@ -37,6 +37,15 @@ EVP_PKEY_gettable_params() returns a constant list of I<params> indicating
 the names and types of key parameters that can be retrieved.
 See L<OSSL_PARAM(3)> for information about parameters.
 
+An B<OSSL_PARAM> of type B<OSSL_PARAM_INTEGER> or
```

Comment:
Should this be in the notes section?  It seems slightly out of place here.  It's also more likely to be read if here.

Reply:
Yeah, I would have this in NOTES

Reply:
I'd prefer to keep it here. As @paulidale says it is much more likely to be read here, and I think this is critical information for understanding how these functions work.